src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx

```diff
@@ -5,6 +5,28 @@ import React from 'react';
 import NameTag from 'people/utils/NameTag';
 import MobileView from '../CodingBounty';
 
+interface Props {
+  isDisabled: boolean;
+  buttonText: string;
+}
+
+const ImageButton = ({ isDisabled, buttonText }: Props) => (
+  <div
+    role="button"
+    style={{
+      width: '117px',
+      height: '40px'
+    }}
+  >
+    <div className="leadingImageContainer">
+      <img className="buttonImage" src="/static/Delete.svg" alt="h" />
+    </div>
+    <button disabled={isDisabled} className="euiText ButtonText css-g2xc3e-euiText-m">
+      {buttonText}
+    </button>
+  </div>
+);
```

Comment:
why are we creating a new component in the test and then testing it this doesn't not test the functionality of CodingBounty at all